### DISCUSSION TITLE
Modern React

### DISCUSSION THREAD

**Discussion by codewithmuh** [community/community#28646] [Other Feature Feedback, Questions, & Ideas]
Who is learning React in 2022?

**Comment by DanilTezin** [comment]
All and sundry

**Comment by Sagar-Sharma-7** [comment]
I will start learning React after Octobers

**Comment by cosmicqbit** [comment]
I'm currently learning React.

**Reply by wTiger0605** [reply]
How long have you been learning React?

**Comment by luisestebanas** [comment]
I am. ✌️

**Comment by Dmitriy-Tyukh** [comment]
I will start learning React

**Reply by wTiger0605** [reply]
What would you learn React from?

**Reply by wTiger0605** [reply]
When?

**Comment by leolanese** [comment]
When I was starting with React this really helps me: ["A complete-intro-to-react-v6"](https://btholt.github.io/complete-intro-to-react-v6/) I hope this will help someone else as well

**Comment by DSDmark** [comment]
I learn **reactjs** a year ago, **react** such an awesome library 
but really I spend a lot of time to figure out about the **render** 
![index](https://user-images.githubusercontent.com/85744816/199485642-249f2cae-442a-4d91-b398-2af21baac864.jpg)

I honestly say, if you take 2 hours in a day then you learn **react** in 6 months, but if you want to advance you need at least 1 year in my case.
